app/models/setting.rb

```diff
@@ -13,4 +13,22 @@ def self.get(key)
     value_object = Setting.where(key: key, status: "Active").first
     value_object.value if value_object
   end
+
+  def active?
+    self.status == 'Active'
+  end
+
+  def self.find_or_create_upcoming_events_setting(options = {})
```

Comment:
I worry about this, because this direction will require one of these for every setting. To a large degree, I don't care if the setting is in the db. The biggest reason to always have them there is to let people know `Hey, there's an adjustment you can make here...`. We could add it to `seeds.rb`, which would make all this move there, keeping the model smaller.

Reply:
I was also doubtful about this, moving it into the seeds is correct option.